Makefile

```diff
@@ -30,6 +30,13 @@ static-code-analysis: vendor ## Runs a static code analysis with phpstan/phpstan
 	mkdir -p .build/phpstan
 	vendor/bin/phpstan analyse --configuration=phpstan.neon
 
+.PHONY: static-code-analysis-baseline
+static-code-analysis-baseline: vendor ## Generates a baseline for static code analysis with phpstan/phpstan
+	mkdir -p .build/phpstan
+	sed -e '/phpstan-baseline\\.neon/ s/^/#/' phpstan.neon > phpstan-without-baseline.neon
```

Comment:
hah, another good workaround.
I did this just last week when i was proud of at the time: https://gist.github.com/bendavies/37a71c8aeb325acc566292ab1ec0fe27
but then ondrejmirtes pointed out to me that really all you need to do is empty the phpstan-baseline.neon before regenerating it, rather than removing the include.

Reply:
Ha, that's actually a good idea!